editor/src/clj/editor/app_view.clj

```diff
@@ -1924,14 +1924,14 @@ If you do not specifically require different script states, consider changing th
 (defn- handle-focus-owner-change! [app-view app-scene new-focus-owner]
   (let [old-editor-tab-pane (g/node-value app-view :active-tab-pane)
         new-editor-tab-pane (editor-tab-pane new-focus-owner)]
-    (when (and (some? new-editor-tab-pane)
-               (not (identical? old-editor-tab-pane new-editor-tab-pane)))
+    (when (some? new-editor-tab-pane)
+      (when (not (identical? old-editor-tab-pane new-editor-tab-pane))
+        (ui/add-style! old-editor-tab-pane "inactive")
+        (ui/remove-style! new-editor-tab-pane "inactive")
+        (g/set-property! app-view :active-tab-pane new-editor-tab-pane))
       (let [selected-tab (ui/selected-tab new-editor-tab-pane)
             resource-node (tab->resource-node selected-tab)
             view-type (tab->view-type selected-tab)]
-        (ui/add-style! old-editor-tab-pane "inactive")
-        (ui/remove-style! new-editor-tab-pane "inactive")
-        (g/set-property! app-view :active-tab-pane new-editor-tab-pane)
         (on-selected-tab-changed! app-view app-scene selected-tab resource-node view-type)))))
```

Comment:
When we remove the inactive tab-pane [here](https://github.com/defold/defold/pull/10476/files#diff-4094da286709d7f8895e995fa84b2aa81727ed5610ff8fcbfb8ceee4f831e699R1919), the selected tab seems to be left in an invalid state. Calling `on-selected-tab-changed!` seems to fix the issue. We could add this after the pane removal, but since we also call `.requestFocus`, we are going to end up here. The `new-editor-tab-pane` will be the same with the `old-editor-tab-pane`, so this change ensures that `on-selected-tab-changed!` will always get called.